app/controllers/gobierto_admin/gobierto_common/custom_fields/custom_fields_controller.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+module GobiertoAdmin
+  module GobiertoCommon
+    module CustomFields
+      class CustomFieldsController < BaseController
+        before_action :check_permissions!
+        before_action :check_class, except: :create_option
+
+        def index
+          set_class_name
+
+          @class_human_name = @class_name.constantize.model_name.human(count: 2)
+          @custom_fields = ::GobiertoCommon::CustomField.where(class_name: @class_name)
```

Comment:
Shouldn't this be filtered by `current_site`? I can see custom fields from other sites in this page.